### ISSUE TITLE
Link to install file down?

### ISSUE THREAD

**Issue by MichiFr** [SAP/abap-cleaner#356]
I've set up a new ADT installation on my Windows machine and used the link https://sap.github.io/abap-cleaner/updatesite to install the cleaner within Eclipse' _Install New Software_ menu.

However, I get this error 
![image](https://github.com/user-attachments/assets/1a960648-99ed-439a-9cfc-630909519e17)

Is the site currently down?

**Comment by sratz** [comment]
The site is up and working.

Could you post the full error message including stack trace if possible from the linked error log?

**Comment by MichiFr** [comment]
Hi and thanks for replying.

All is running OK again on my side.

The issue raised because of some undefined order of installing Eclipse 2024 - 06 and some of its plugins, including SAP ADT und Abap Cleaner. 

During this installation I did an additional update of Eclipse to version 2024-09 and used old workspaces. 

Any circumstances then destroyed the whole installation and no update and no installation of further updates were possible any more. Even Eclipse marketplace failed completely. 

I then swiped the complete installation and workspaces and restarted from scratch. 

Now ot works perfectly well, all update site are up and running. 

Thanks for reading! 
Michael
